em++

```diff
@@ -9,8 +9,9 @@ sys.argv += ['--emscripten-cxx']
 
```

Comment:
we could do the `if __name__ == '__main__'` once on the top level, then put the `if sys.version` inside that? same for the other two scripts.

Unless the `import emcc` has to run even in the non-`__main__` case?

lgtm either way, with that change if possible, or without it if windows won't work with it.